sdcm/sct_events/base.py

```diff
@@ -296,9 +353,11 @@ class ContinuousEvent(SctEvent, abstract=True):
     _duration: Optional[int] = None
 
     def __init__(self,
+                 node: str = "",
                  severity: Severity = Severity.UNKNOWN,
                  publish_event: bool = True):
         super().__init__(severity=severity)
+        self.node = node
```

Comment:
let's move it to `ScyllaDatabaseContinuousEvent`

Reply:
Fixed